# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chkrootkit/make_install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- chkrootkit-0.47/Makefile	2006-10-09 21:29:19.000000000 +0200
+++ chkrootkit-0.47-fixed/Makefile	2007-06-17 14:31:42.133410484 +0200
@@ -7,6 +7,8 @@
 CFLAGS	 = -DHAVE_LASTLOG_H
 STATIC   = -static
 
+prefix = /usr
+
 ###
 ### Solaris 2.x
 ###
@@ -41,36 +43,32 @@
 
 chklastlog:   chklastlog.c
 	${CC} ${CFLAGS} -o $@ chklastlog.c
-	@strip $@
 
 chkwtmp:   chkwtmp.c
 	${CC} ${CFLAGS} -o $@ chkwtmp.c
-	@strip $@
 
 ifpromisc:   ifpromisc.c
 	${CC} ${CFLAGS} ${LDFLAGS}  -D_FILE_OFFSET_BITS=64 -o $@ ifpromisc.c
-	@strip $@
 
 chkproc:   chkproc.c
 	${CC} ${LDFLAGS} -o $@ chkproc.c
-	@strip $@
 
 chkdirs:   chkdirs.c
 	${CC} ${LDFLAGS} -o $@ chkdirs.c
-	@strip $@
 
 check_wtmpx:   check_wtmpx.c
 	${CC} ${LDFLAGS} -o $@ check_wtmpx.c
-	@strip $@
 
 chkutmp:   chkutmp.c
 	${CC} ${LDFLAGS} -o $@ chkutmp.c
-	@strip $@
 
 
 strings-static:   strings.c
 	${CC} ${STATIC} ${LDFLAGS} -o $@ strings.c
-	@strip $@
 
 clean:
 	rm -f ${OBJS} core chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static chkutmp
+
+install: sense
+	cp -vf chkrootkit $(DESTDIR)$(prefix)/sbin/
+	cp -vf chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static chkutmp $(DESTDIR)$(prefix)/sbin/
